NEWS.md

```diff
@@ -1,4 +1,22 @@
+## 13.7.0 - Unreleased
```

Comment:
folio-kafka-wrapper is updated already, so you can use release version now

Reply:
Updated